packages/render-fidelity-tools/package.json

```diff
@@ -36,7 +36,7 @@
     "@types/puppeteer": "^5.4.6",
     "@types/rimraf": "^3.0.1",
     "@types/three": "^0.156.0",
-    "filament": "1.31.5",
+    "filament": "1.44.0",
```

Comment:
Did none of the existing renders change appreciably when these versions were bumped?

Reply:
Okay, lots of PRs merged - let's see how this does when it's all pulled together!